app/scripts/modules/core/src/pipeline/config/triggers/pubsub/pubsub.trigger.ts

```diff
@@ -0,0 +1,44 @@
+import {IController, module} from 'angular';
```

Comment:
This isn't done consistently, but the style for Deck to include spaces between the braces and the imports:
```
import { IController, module } from 'angular';
```

I can show you how to set this as the default in IntelliJ if you want.